config/rbac/role.yaml

```diff
@@ -67,14 +67,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
```

Comment:
this section is unneeded as it is duplicated from line [107](https://github.com/aws/amazon-cloudwatch-agent-operator/pull/63/files#diff-6d4c4e29ec096149781536bb9118fde8242c1901df764e3eda3019ac8503d6c7R99)